components/alert/alert.hbs

```diff
@@ -210,4 +210,74 @@
 			</div>
 		</div>
 	</div>
+
+	<h2>Dark Theme alert without a checkbox - With image</h2>
+	<h3>This checkbox will remain until the page is refreshed</h3>
+	<div class="row">
+		<div class="col-12">
+			<div class="d-none msds-alert msds-alert--dark-theme msds-bg-gray-2" data-alert-id="10">
+				<div class="msds-alert__close-click-area">
+					<svg class="msds-alert__close-icon">
+						<use href="{{ path '/msds-spritemap.svg' }}#cross" />
+					</svg>
+				</div>
+				<div class="msds-alert__content-container">
+					<div class="msds-alert__image msds-innovation msds-rich-icon"></div>
+					<div class="msds-alert__text-container">
+						<div class="msds-alert__header">
+							AGG Solution
+						</div>
+						<p class="msds-alert__text">
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit
+							amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie.
+							Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac
+							porttitor pulvinar.
+						</p>
+						<div class="msds-alert__action-container">
+							<button type="button"
+								class="msds-btn msds-btn--secondary-dark msds-btn--sm">ok,
+								i get it</button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<h2>Dark Theme alert - Solution alert, without a checkbox - With image</h2>
```

Comment:
I would suggest making a "light themed alert" that uses the badge/tag and possibly also looking at creating documentation for the alerts if you have time, or create a UST that can be prioritized in an upcoming sprint so we can get MSDS a little more clean

Reply:
Added a light theme solution alert.

ill create a ust for documentation